src/pages/api/[...params].ts

```diff
@@ -0,0 +1,33 @@
+import axios, { isAxiosError } from "axios";
+import { NextApiHandler } from "next";
+
+const handler: NextApiHandler = async (req, res) => {
+  if (process.env.NEXT_PUBLIC_API_URL === undefined) return res.status(500).json({ message: "Missing API URL" });
+  const token = req.cookies.token;
+  const url = `${process.env.NEXT_PUBLIC_API_URL}${req.url?.substring(4) || ""}`;
```

Comment:
what does this do

Reply:
`req.url` is for example `/api/friends/list`. The substring will remove the `/api` part, and appends it to the backend API url, so it will become `https://api.testaustime.fi/friends/list`. Guess I could make it clearer...

Reply:
That would be awesome